Source/Mosa.VisualStudio.ProjectTemplate/Mosa.Starter.x86.csproj

```diff
@@ -5,23 +5,47 @@
 		<OutputType>Exe</OutputType>
 		<StartupObject>$safeprojectname$.Boot</StartupObject>
 		<DebugType>full</DebugType>
-		<LangVersion>8.0</LangVersion>
+		<LangVersion>9.0</LangVersion>
 		<OutputPath>bin\</OutputPath>
 		<AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
 		<DisableStandardFrameworkResolution>true</DisableStandardFrameworkResolution>
 		<DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
+		<UseAppHost>false</UseAppHost>
 		<AllowUnsafeBlocks>true</AllowUnsafeBlocks>
 	</PropertyGroup>
 	<ItemGroup>
-		<PackageReference Include="Mosa.Korlib" Version="*" />
-		<PackageReference Include="Mosa.Kernel.x86" Version="*" />
-		<PackageReference Include="Mosa.Platform.x86" Version="*" />
-		<PackageReference Include="Mosa.Runtime" Version="*" />
-		<PackageReference Include="Mosa.Tools.Package" Version="*" />
+		<Reference Include="Mosa.DeviceDriver">
+		  <HintPath>C:\Program Files (x86)\MOSA-Project\bin\Mosa.DeviceDriver.dll</HintPath>
+		</Reference>
+		<Reference Include="Mosa.DeviceSystem">
+		  <HintPath>C:\Program Files (x86)\MOSA-Project\bin\Mosa.DeviceSystem.dll</HintPath>
+		</Reference>
+		<Reference Include="Mosa.FileSystem">
+		  <HintPath>C:\Program Files (x86)\MOSA-Project\bin\Mosa.FileSystem.dll</HintPath>
+		</Reference>
+		<Reference Include="Mosa.Kernel.x86">
+		  <HintPath>C:\Program Files (x86)\MOSA-Project\bin\Mosa.Kernel.x86.dll</HintPath>
+		</Reference>
+		<Reference Include="Mosa.Platform.Intel">
```

Comment:
Two issues: 

1. exclude compiler only classes (MOSA.Platform.*)

2. Path should not be hard coded.

Reply:
But how can we find the DLls then if we don't hardcode it?

Reply:
via the package!

Reply:
But I don't think the packages are recompiled every time, so you would use outdated versions of the packages, right? Maybe there's something I don't understand here. Also, it didn't work with the packages before, for some reason.

Reply:
This is complicated - since we attempt to use the packages to keep the starter project and compiler tool chain up to date. However, it is far from stable and still very experimental. So as such, I'm okay if it breaks.